app/controllers/timelogs_controller.rb

```diff
@@ -43,7 +46,21 @@ def destroy
 
   private
     def timelogs_params
-      params.require(:timelog).permit(:start_time, :duration, :user_id)
+      params.require(:timelog).permit(:start_time, :duration, :user_id, :trello_card)
+    end
+
+    def load_user_trello
+      if current_user.has_trello?
+        Trello.configure do |config|
```

Comment:
This should not live in controller.
This should go at least to service object
https://blog.engineyard.com/2014/keeping-your-rails-controllers-dry-with-services